src/notebooks/pyproject.toml

```
@@ -11,6 +11,7 @@ matplotlib = "^3.3.2"
 pandas = "^1.1.3"
 ipykernel = "^5.3.4"
 jupyter = "^1.0.0"
+edfi-lms-file-utils = "^1.0.0-alpha.0"

```

```suggestion
edfi-lms-file-utils = "1.0.0a.1"
```

I am surprised this would work with "alpha" in the name, as the system actually publishes to pypi as "1.0.0a0". Brad pointed out that we should probably hard-code to a specific prerelease instead of using the carat `^` notation, which means "use latest 1.0.x", because we might create different pre-releases at the same time and that could cause integration testing problems.

---

But don't make this change yet, as I will use it to test my workflow change above.